### PR TITLE
OPSRC-496/Add bitbag coding standard and ecs fix 

### PR DESCRIPTION
**Pull request by fabulousPuppet**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -33,14 +34,13 @@
...
-        "sylius-labs/coding-standard": "^4.0",
         "symfony/browser-kit": "^4.4 || ^5.2",
         "symfony/debug-bundle": "^4.4 || ^5.2",
         "symfony/dotenv": "^4.4 || ^5.2",
         "symfony/intl": "^4.4 || ^5.2",
         "symfony/web-profiler-bundle": "^4.4 || ^5.2",
-        "symplify/easy-coding-standard": "^8.3",
-        "vimeo/psalm": "4.6.4"
+        "symplify/easy-coding-standard": "^9.4",
+        "vimeo/psalm": "4.24.0"
```
Comment (senghe) [inline composer.json:43]: Why this changes?

Comment (fabulousPuppet) [reply composer.json:43]: These packages had too low versions and the update was required during installation of "bitbag/coding-standard".

**Review by senghe — APPROVED** [review]
